python/python/tests/test_async_client.py

```diff
@@ -2124,74 +2124,126 @@ async def test_zinterstore(self, redis_client: TRedisClient):
         assert await redis_client.zadd(key1, members_scores1) == 2
         assert await redis_client.zadd(key2, members_scores2) == 3
 
+        # zinter tests
+        zinter_map = await redis_client.zinter([key1, key2])
+        expected_zinter_map = ["one", "two"]
+        assert zinter_map == expected_zinter_map
+
```

Comment:
Add new functions to cross slot test
https://github.com/aws/glide-for-redis/blob/d2fabc19dc3f403366ee8fa50d07c968531ff5e9/python/python/tests/test_async_client.py#L3542

Reply:
Addressed